### ISSUE TITLE
Error message not clear when .taskcluster.yml fails json schema validation

### ISSUE THREAD

**Issue by petemoore** [taskcluster/taskcluster-github#142]
Getting this message [here](https://github.com/taskcluster/taskcluster-worker/pull/155#commitcomment-20834871).

CC @owlishDeveloper @djmitche @imbstack

**Comment by djmitche** [comment]
```
Feb 10 10:38:40 taskcluster-github app/worker.1: Fri, 10 Feb 2017 15:38:40 GMT taskcluster-github:handlers:fe36e980-efa6-11e6-8f05-5cbda96b5004 .taskcluster.yml was not formatted correctly. Leaving comment on Github. 
```

The task is missing `task.github.extra`.

**Comment by petemoore** [comment]
Ah, thanks! I guess that is my fault then. :-)

Still, it might not be a bad idea to perform a json schema validation, and report and validation errors, so that the root cause can be better understood - I'll update the issue title... Thanks Dustin!

**Comment by djmitche** [comment]
Fixed in #143

**Comment by djmitche** [comment]
@petemoore just for the record: the JSON was valid, it just didn't create that task, so the change was to not raise an exception in that case.  `.taskcluster.yml` is not specific to TC-github (it is used by mozilla-taskcluster, too, for example), so the presence of `extra.github` is used to indicate tasks which that service should run.
